test/ng/directive/ngOptionsSpec.js

```diff
@@ -2515,37 +2521,6 @@ describe('ngOptions', function() {
       expect(element.find('option')[1].selected).toBeTruthy();
     });
 
-    it('should not write disabled selections from model', function() {
```

Comment:
This one :) It's a duplicate of https://github.com/angular/angular.js/pull/14233/files#diff-2cda0d7a6d8ec47da3560fbe59a634e3R850